src/toTransform.js

```diff
@@ -0,0 +1,151 @@
+(function () {
+  function InvalidArgument (message) {
+    this.message = message;
+    this.name = 'InvalidArgument';
+  }
+
+  function isNumeric (number) {
+    return !isNaN(number);
+  }
+
+  function convertTranslate (properties) {
```

Comment:
These conversion functions should just take their respective property's value. They should operate independently from the other properties.